### PR TITLE
more linked features!

### PR DESCRIPTION
**Pull request by eleftherioszisis**

### CODE REVIEW

File: neurom/features/tests/test_neuron_features.py [hunk 1]
```diff
@@ -2,19 +2,78 @@
...
+NEURONS = [NEURON, NEURON]
+
+def test_soma_radii():
+    nt.eq_(list(nf.soma_radii(NEURONS)), [0.17071067811865476, 0.17071067811865476])
+
+def test_soma_surface_areas():
+    area = 4. * math.pi * list(nf.soma_radii(NEURON))[0] ** 2
+    nt.eq_(list(nf.soma_surface_areas(NEURONS)), [area, area])
+
+def test_trunk_origin_elevations():
```
Comment (juanchopanza) [inline neurom/features/tests/test_neuron_features.py]: Could you test empty sets too? i.e. use a tree type that isn't present in the neuron.

File: neurom/features/tests/test_neuron_features.py [hunk 2]
```diff
@@ -2,19 +2,78 @@
...
+
+    t2 = Tree((0, -1, 0, 2))
+    t2.type = TreeType.basal_dendrite
+    n1.neurites = [t2]
+    n1.soma = s
+
+    pop = [n0, n1]
+    nt.eq_(list(nf.trunk_origin_elevations(pop)), [0.0, np.pi/2., -np.pi/2.])
+
+def test_trunk_origin_azimuths():
```
Comment (juanchopanza) [inline neurom/features/tests/test_neuron_features.py]: Same comment re. empty set.

**Comment by lidakanari** [comment]
I think we also need:
section_radial_distances
section_path_distances

Check the neurom.sections module :)

**Comment by juanchopanza** [comment]
I am OK with this if you're done. The other features can go in a future pull request.

**Comment by eleftherioszisis** [comment]
Yep, you can merge it and I will make a new one when I put them in place.

**Pull request merged**